src/libraries/Microsoft.PowerFx.Connectors/Tabular/CdpTableResolver.cs

```diff
@@ -38,30 +38,32 @@ public CdpTableResolver(CdpTable tabularTable, HttpClient httpClient, string uri
             Logger = logger;
         }
 
-        public async Task<ConnectorType> ResolveTableAsync(string tableName, CancellationToken cancellationToken)
+        public async Task<ConnectorType> ResolveTableAsync(string logicalName, CancellationToken cancellationToken)
         {
-            // out string name, out string displayName, out ServiceCapabilities tableCapabilities
+            // out string name, out string displayName, out ServiceCapabilities tableCapabilities            
             cancellationToken.ThrowIfCancellationRequested();
 
+            // if we have the list of table, we can convert from display name to logical name
             if (_tabularTable.Tables != null)
             {
-                RawTable t = _tabularTable.Tables.FirstOrDefault(tbl => tbl.DisplayName == tableName);
+                RawTable t = _tabularTable.Tables.FirstOrDefault(tbl => tbl.DisplayName == logicalName);
```

Comment:
>tbl.DisplayName == logicalName [](http://example.com/codeflow?start=72&length=30)

Although this suggests that "logicalOrDisplayName" may be a better name for parameter ;)